### PR TITLE
Dark Mode: Send flow inputs

### PR DESCRIPTION
**Pull request by darkwing**
TODO: Still needs the swap icon to be light

SendySend

### CODE REVIEW

File: ui/css/itcss/components/send.scss [hunk 1]
```diff
@@ -720,9 +712,9 @@
       position: absolute;
       height: 220px;
       width: 100%;
-      border: 1px solid var(--geyser);
+      border: 1px solid var(--color-border-default);
```
Comment (GuillaumeRx) [inline ui/css/itcss/components/send.scss:715]:
```suggestion
      border: 1px solid var(--color-border-muted);
```

File: ui/css/itcss/components/send.scss [hunk 2]
```diff
@@ -737,7 +729,7 @@
     }
 
     &__input-wrapper {
-      border: 1px solid var(--alto);
+      border: 1px solid var(--color-border-default);
```
Comment (GuillaumeRx) [inline ui/css/itcss/components/send.scss:732]:
```suggestion
      border: 1px solid var(--color-border-muted);
```

**Review by GuillaumeRx — CHANGES_REQUESTED** [review]

**Comment by darkwing** [comment]
Updated!

**Review by GuillaumeRx — DISMISSED** [review]
LGTM !

**Review by georgewrmarshall — APPROVED** [review]

**Review by GuillaumeRx — APPROVED** [review]

**Review by NiranjanaBinoy — APPROVED** [review]

**Pull request merged**